test/mochitest/browser_dbg-inspector-integration.js

```diff
@@ -24,4 +25,23 @@ add_task(async function() {
   findElementWithSelector(dbg, "button.open-inspector").click();
 
   await waitForInspectorPanelChange(dbg);
-});
\ No newline at end of file
+});
+
+add_task(async function() {
+  const dbg = await initDebugger("doc-event-handler.html");
+
+  invokeInTab("synthesizeClick");
+  await waitForPaused(dbg);
+
+  findElement(dbg, "frame", 2).focus();
+  await clickElement(dbg, "frame", 2);
+
+  // Hover over the token to launch preview popup
+  await tryHovering(dbg, 5, 8, "popup");
+
+  // Click the first inspector buttom to view node in inspector
+  await waitForElementWithSelector(dbg, "button.open-inspector");
+  findElementWithSelector(dbg, "button.open-inspector").click();
```

Comment:
nit: it would be nice to name this selector so that we can re-use it in other tests and if the class name changes change it in one place